.circleci/config.yml

```diff
@@ -15,16 +15,16 @@ commands:
       - restore_cache:
           name: Restore Ruby Package Cache
           keys:
-            - v3-gem-cache-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
-            - v3-gem-cache-{{ arch }}-{{ .Branch }}-
-            - v3-gem-cache-{{ arch }}-
+            - v5-gem-cache-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+            - v5-gem-cache-{{ arch }}-{{ .Branch }}-
+            - v5-gem-cache-{{ arch }}-
       - run:
           name: Install base Ruby dependencies
           command: |
-            bundle install --jobs=4 --retry=3 --path=vendor/bundle
+            bundle install --jobs=1 --retry=3 --path=vendor/bundle
```

Comment:
This seems to stop the debase install blowing up issue, did a manual cache bump to test it. Could be a fluke. 
https://app.circleci.com/pipelines/github/Gusto/apollo-federation-ruby/538/workflows/27745a84-b5c5-4e6a-9bb9-e5ec71f1cc0c/jobs/3060

Either way, while we're trying to figure out bundler issues, I thought it's wise now to limit bundling to a job to avoid any concurrency issues.